testing/utils/project_template.cmake.in

```diff
@@ -49,5 +49,5 @@ add_subdirectory(${rapids-cmake_SOURCE_DIR} ${rapids-cmake_BINARY_DIR})
 
 # We need to enable at least one language so that we get properly
 # generated system search paths.
-project(@test_name@ LANGUAGES CXX)
+project(@test_name_stem@ LANGUAGES CXX)
```

Comment:
Did we not have any tests where this bug manifested before?

Reply:
It didn't actually create any problems, which surprised me! I just noticed that the generated file was wrong.